int32 McStatus
string ItemName
geometry_msgs/Pose InterfaceJoint
#float32[] InterfaceJointLocation
#float32[] InterfaceJointOrientation
